src/ngraph/runtime/gpu/gpu_external_function.cpp

```diff
@@ -36,6 +36,8 @@
 #include "ngraph/op/add.hpp"
 #include "ngraph/op/allreduce.hpp"
 #include "ngraph/op/and.hpp"
+#include "ngraph/op/argmax.hpp"
```

Comment:
Maybe it is better to use the same auto generation procedure for all headers related to particular operation?

Reply:
I did think about that but the class names are CamelCase and the header files are snake_case and conversion inside a macro is difficult/impossible and I was not that interested in trying.